ablator/utils/base.py

```diff
@@ -95,6 +219,14 @@ def parse_device(device: ty.Union[str, list[str]]):
 
 
 def init_weights(module: nn.Module):
+    """
+    Initialize the weights of a module.
```

Comment:
- If module is a Linear layer, initialize weight values to be coming from a normal distribution N(mu=0,std=1.0). Biases, if available, are initialized to zeros
- If module is an Embedding layer, initialize embeddings such that values in each embedding comes from N(mu=0,std=1.0). If padding is enabled, set padding embedding to a zero vector
- If module is a Normalization layer, set all biases to zeros and all weights to 1